closed/src/java.base/share/classes/com/sun/crypto/provider/NativeGaloisCounterMode.java

```diff
@@ -353,6 +353,10 @@ int encryptFinal(byte[] in, int inOfs, int len, byte[] out, int outOfs)
             throw new ShortBufferException("Output buffer too small");
         }
 
+        // avoid any race conditions in OpenSSL when the key/iv are used concurrently
+        byte[] keyCopy = key.clone();
+        byte[] ivCopy = iv.clone();
```

Comment:
These copies may not be consistent, synchronization may be required.

Reply:
I moved these inside the synchronized block